src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  &__container {
+    width: 80vmin;
+    height: 80vmin;
+    border: dotted 1vmin #000;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__minute,
+  &__seconds,
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    transform-origin: bottom center;
+  }
+
+  &__minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    animation: rotate-minute 3600s infinite linear;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    animation: rotate-second 60s infinite linear;
+  }
+
+  &__center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  @keyframes rotate-minute {
+    from {
+      transform: translate(-50%, -100%) rotate(0deg);
+    }
+    to {
+      transform: translate(-50%, -100%) rotate(360deg);
+    }
+  }
+
+  @keyframes rotate-second {
+    from {
+      transform: translate(-50%, -100%) rotate(0deg);
```

Comment:
Similar to the `rotate-minute` animation, the `transform` property in the keyframes for `rotate-second` should only include the rotation. Remove the `translate(-50%, -100%)` from the keyframes as it's already applied in the element's style.